src/core/qgstemporalproperty.h

```diff
@@ -59,6 +68,11 @@ class CORE_EXPORT QgsTemporalProperty : public QObject
     */
     bool isActive() const;
 
+    /**
+     * Returns flags associated to the temporal property.
+     */
+    virtual const QgsTemporalProperty::Flags flags() const { return nullptr }
```

Comment:
Can you still drop the initial const here?

Reply:
@nyalldawson , should I keep the const on overridden functions?

Reply:
Leave the second const, drop the first on all